setup.cfg

```diff
@@ -49,10 +49,13 @@ python_requires = >= 3.9
 
 [options.extras_require]
 akafka =
-    aiokafka~=0.7.2
+    aiokafka>=0.7.2
```

Comment:
Maybe limit major version here and below with `^=`?

Reply:
I don't find "^=" to be documented anywhere, could you provide a resource that explains it? In this PEP, I would `~=` but not `^=`.

Reply:
Sorry, scratch that. I was in JavaScript land (https://semver.npmjs.com/) when I wrote this. Python has only `~=`.